vowpalwabbit/cb_explore_adf_squarecb.cc

```diff
@@ -280,6 +289,17 @@ void cb_explore_adf_squarecb::predict_or_learn_impl(VW::LEARNER::multi_learner&
   }
 }
 
+void cb_explore_adf_squarecb::save_load(io_buf& io, bool read, bool text)
+{
+  if (io.num_files() == 0) { return; }
+  if (!read || _model_file_version >= VERSION_FILE_WITH_CB_TO_CBADF)
```

Comment:
Please add a new constant that represents square cb serializing and deserializing the counter. It's fine for it to use the same verison but should have its own define.